bots/pr/src/main/java/org/openjdk/skara/bots/pr/CSRCommand.java

```diff
@@ -34,7 +34,7 @@
     private static final String CSR_LABEL = "csr";
 
     private static void showHelp(PrintWriter writer) {
-        writer.println("usage: `/csr [unneeded]`, requires that the issue the pull request links to an approved [CSR](https://wiki.openjdk.java.net/display/csr/Main) request.");
+        writer.println("usage: `/csr [needed|unneeded]`, requires that the issue the pull request links to an approved [CSR](https://wiki.openjdk.java.net/display/csr/Main) request.");
```

Comment:
Looks like a pre-existing typo:
```suggestion
        writer.println("usage: `/csr [needed|unneeded]`, requires that the issue the pull request links to, links to an approved [CSR](https://wiki.openjdk.java.net/display/csr/Main) request.");
```

Reply:
Assuming the requirement is PR -(links to)-> Issue -(links to)-> CSR

Reply:
Thanks for noticing this, I changed the usage message in a970e4ce2027a8f35d7e9dfffdc072e7c472b369. I used "... the issue the pull request refers to links to an approve" in order to not get to consecutive "links to" :smiley_cat: What do you think?

Reply:
Looks good 👍

Reply:
:+1: